arcade/experimental/sprite_depth_cosine.py

```diff
@@ -88,11 +87,10 @@ def on_key_press(self, symbol: int, modifiers: int):
             self.text_use_depth.text = f"SPACE: Toggle depth testing ({self.use_depth})"
 
     def on_update(self, delta_time):
-        self.time += delta_time
-
         for i, sprite in enumerate(self.sprite_list):
+            # time ticks automatically now due to Window's global clock
```

Comment:
We have to assume most users will have never looked at this example before so this comment doesn't make sense. (Also this might not have existed pre 3.0 so even less likely to have been seen before)

Reply:
Removed.